cmd/manager/main.go

```diff
@@ -96,22 +91,23 @@ func main() {
 	log.Print("Starting the Cmd.")
 	stopCh := signals.SetupSignalHandler()
 
+	// monitorDoneCh will recieve a signal when threads have stopped updating ClusterOperator status
```

Comment:
`monitorDoneCh!=statusReportingDoneCh`... will address later.

Reply:
I don't understand this comment and the naming of the variables. Please elaborate.

Reply:
Sure.

1. I named the channel `statusReportingDoneCh` because the channel is closed when the go routine responsible for reporting `ClusterOperator` Status successfully updates the operator and exits.
2. The comment should be updated to better capture the intent. Perhaps something like:
`// statusReportingDoneCh will be closed after the operator has successfully stopped reporting ClusterOperator status.`

Reply:
Please reflect this and this PR should be good to be merged. You just need to clearly explain what you are doing now in the code and describe how and why you are moving away from multiple channels in the commit message.

Reply:
Will do!